packages/auto-approve/src/check-config.ts

```diff
@@ -14,11 +14,12 @@
 
 // This file manages the logic to check whether a given config file is valid
 
-// eslint-disable-next-line node/no-extraneous-import
 import yaml from 'js-yaml';
 import Ajv from 'ajv';
 import {Octokit} from '@octokit/rest';
 import {Configuration} from './auto-approve';
+// eslint-disable-next-line node/no-extraneous-import
+import {RequestError} from '@octokit/request-error';
```

Comment:
It's weird that eslint would see this as an extraneous import, if you're actually using it below O_o